### DISCUSSION TITLE
Which firmware to download ? The appropriate firmware isn't automatically presented

### DISCUSSION THREAD

**Discussion by Nico128** [pvvx/ATC_MiThermometer#590] [Q&A]
Hi!

I've a LYWSD03MMC that I flashed with ATC_Thermometer.bin from atc1441/ATC_MiThermometer. 
Then I red on Guthub's atc1441 that he recommands to use the firmware from pvvx/ATC_MiThermometer.

When I connect to the device, it says "custom firmware detected, please use https://pvvx.github.io/ATC_MiThermometer/TelinkMiFlasher.html".
Ok. Then I must select a firmware. It isn't automatically presented.

Which one must I download ?

Thank you!

**Comment by pvvx** [comment]
https://github.com/pvvx/ATC_MiThermometer/raw/refs/heads/master/ATC_v49.bin

This link can be pasted into the file selection menu.

**Reply by Nico128** [reply]
Thank you very much! It works
